src/domain/timeline/calendar/components/CalendarStyles.ts

```diff
@@ -169,6 +179,13 @@ const CalendarStyles = styled(Box)(({ theme }) => ({
     backgroundColor: 'transparent',
     zIndex: 3,
   },
+  // Enhanced keyboard focus visibility
+  '.react-calendar button:focus-visible': {
```

Comment:
It's not this line, but I wanted to leave a comment here in CalendarStyles. Very good job, I like it!
The only thing I've seen, and I think it's a mistake, on hover the Add event button fades to grey:
<img width="314" height="310" alt="image" src="https://github.com/user-attachments/assets/5b14eaa3-845a-4742-8303-d8edf24deb9f" />
<img width="314" height="310" alt="image" src="https://github.com/user-attachments/assets/17a55e75-91a5-4269-a2d4-ad08deed9554" />